engine/src/main/java/org/operaton/bpm/engine/impl/json/ModificationCmdJsonConverter.java

```diff
@@ -38,33 +38,37 @@ public class ModificationCmdJsonConverter extends JsonObjectConverter<AbstractPr
   public static final String CANCEL_ACTIVITY_INSTANCES = "cancelActivityInstances";
   public static final String PROCESS_INSTANCE = "processInstances";
   public static final String CANCEL_TRANSITION_INSTANCES = "cancelTransitionInstances";
+  public static final String ANCESTOR_ACTIVITY_INSTANCE_ID = "ancestorActivityInstanceId";
 
   @Override
   public JsonObject toJsonObject(AbstractProcessInstanceModificationCommand command) {
     JsonObject json = JsonUtil.createObject();
 
-    if (command instanceof ActivityAfterInstantiationCmd instantiationCommand) {
-      JsonUtil.addField(json, START_AFTER, instantiationCommand.getTargetElementId());
-    }
-    else if (command instanceof ActivityBeforeInstantiationCmd instantiationCommand) {
-      JsonUtil.addField(json, START_BEFORE, instantiationCommand.getTargetElementId());
-    }
-    else if (command instanceof TransitionInstantiationCmd instantiationCommand) {
-      JsonUtil.addField(json, START_TRANSITION, instantiationCommand.getTargetElementId());
-    }
-    else if (command instanceof ActivityCancellationCmd cancellationCommand) {
-      JsonUtil.addField(json, CANCEL_ALL, cancellationCommand.getActivityId());
-      JsonUtil.addField(json, CANCEL_CURRENT, cancellationCommand.isCancelCurrentActiveActivityInstances());
-    }
-    else if (command instanceof ActivityInstanceCancellationCmd cancellationCommand) {
-      JsonUtil.addField(json, CANCEL_ACTIVITY_INSTANCES, cancellationCommand.getActivityInstanceId());
-      JsonUtil.addField(json, PROCESS_INSTANCE, cancellationCommand.getProcessInstanceId());
-    }
-    else if (command instanceof TransitionInstanceCancellationCmd canellationCommand) {
-      JsonUtil.addField(json, CANCEL_TRANSITION_INSTANCES, canellationCommand.getTransitionInstanceId());
-      JsonUtil.addField(json, PROCESS_INSTANCE, canellationCommand.getProcessInstanceId());
+    if (command instanceof ActivityAfterInstantiationCmd) {
```

Comment:
We have refactored "instanceof" checks to use the "new" style with direct assignment, I'd prefer if we did this in the backports directly, to not import this old style into our repo again